packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -202,13 +202,25 @@ export const UnitRowTwoStepAuth = ({
         subRows={getSubRows()}
       >
         {exists && verified ? (
-          <FtlMsg id="tfa-row-enabled-description">
-            <p className="text-sm mt-3">
-              Your account is protected by two-step authentication. You will
-              need to enter a one-time passcode from your authenticator app when
-              logging into your Mozilla account.
-            </p>
-          </FtlMsg>
+          <>
+            <FtlMsg id="tfa-row-enabled-description">
+              <p className="text-sm my-2">
+                Your account is protected by two-step authentication. You will
+                need to enter a one-time passcode from your authenticator app
+                when logging into your Mozilla account.
+              </p>
+            </FtlMsg>
+            <div className="mb-4">
+              <FtlMsg id="tfa-row-enabled-info-link">
+                <LinkExternal
+                  href="https://support.mozilla.org/kb/secure-firefox-account-two-step-authentication"
+                  className="link-blue text-sm "
```

Comment:
Nit: extra space

Should we have a TODO here to updating the heading to the proper link when it's created?

Reply:
I think in this case the link is correct as is - this is a link to general information about two-step authentication. Where we use a similar link name specifically for backup recovery phone we could update the link to the section that directly covers that method.